web/instances/checks.go

```diff
@@ -232,7 +232,9 @@ func checkSharings(c echo.Context) error {
 		return wrapError(err)
 	}
 
-	results, err := sharing.CheckSharings(i)
+	skipFSConsistency, _ := strconv.ParseBool(c.QueryParam("SkipFSConsistency"))
```

Comment:
Because we're looking specifically at the FS objects. I wanted to make a difference with the other checks. But if you can think of a better alternative, I'm open to suggestions :).